webservices/legal_docs/current_murs.py

```diff
@@ -285,14 +298,16 @@ def get_documents(case_id, bucket, bucket_name):
                 'text': row['ocrtext'],
                 'document_date': row['document_date'],
             }
-            pdf_key = 'legal/murs/current/%s.pdf' % row['document_id']
-            logger.info("S3: Uploading {} Orig filename: {}".format(pdf_key, document['filename']))
+            pdf_key = 'legal/murs/current/{0}/{1}'.format(row['case_no'],
```

Comment:
As discussed offline, we could lose the `/current`. Also, check archived MURs.

Reply:
@vrajmohan I confirmed archived MURs don't have `/archived` document URLs: https://fec-dev-proxy.app.cloud.gov/files/legal/murs/4772.pdf